.eslintrc

```diff
@@ -72,7 +72,9 @@
     "react/sort-comp": 0,
     "react/button-has-type": 0,
     "flowtype/generic-spacing": 0,
-    "no-param-reassign": ["warn"]
+    "no-param-reassign": ["warn"],
```

Comment:
not a fan of disabling this rule... it is a good rule and can help catch some really nasty bugs.

Reply:
Ok we can change that back... That was already committed to dev however and was not a change introduced by this PR

Reply:
For clarity I am going to revert the changes to this rule in a separate PR